server/service/src/sync/program_schemas/hiv_care_encounter.json

```diff
@@ -389,10 +385,18 @@
           "properties": {
             "bloodPressure": {
               "description": "75367002\tBlood pressure [mmHg]",
-              "examples": ["90/60", "120/80"],
-              "pattern": "^\\d{3}\\/\\d{2}$",
-              "title": "BP in mmHg",
-              "type": "string"
+              "properties": {
+                "diastolic": {
+                  "pattern": "^\\d{3}",
```

Comment:
Pattern doesn't apply for number... You could have maximum: 999 though

Reply:
+ minimum

Reply:
[157a795](https://github.com/openmsupply/open-msupply/pull/1267/commits/157a795849691aaa257927f70da39c85a1b5f58b) (: